docs/en/datasets/recog.md

```diff
@@ -1044,3 +1045,61 @@ should be as follows:
   │   ├── train_label.jsonl
   │   └── val_label.jsonl (optional)
   ```
+
+## HierText
+
+- Step1 (optional): Install [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html). Here is a brief installation procedure, if you encounter problems, please check the official website.
+  ```bash
+  curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+  unzip awscliv2.zip
+  sudo ./aws/install
+  ./aws/install -i /usr/local/aws-cli -b /usr/local/bin
+  !aws configure
+  # this command will require you to input keys, you can skip them except
+  # for the Default region name
+  # AWS Access Key ID [None]:
+  # AWS Secret Access Key [None]:
+  # Default region name [None]: us-east-1
+  # Default output format [None]
```

Comment:
Since there are also other datasets that use AWS CLI, we may create a new section after the Overview section (or somewhere), and link all these steps to that section.

![image](https://user-images.githubusercontent.com/45810070/165885357-ba4ef261-e534-451e-b170-2c98e284de2d.png)

And do not forget to update the link in other dataset that uses the AWS CLI